avaframe/com1DFA/com1DFA.py

```diff
@@ -293,24 +294,25 @@ def runSamos(cfg, avaDir):
 
                 # Create dictionary
                 dictVar = {}
```

Comment:
Maybe rename dictVar dictNull etc to something like reportVar reportNull to make it obvious this is somehow related to the report generation

Reply:
dictionaries renamed to: reportVar, reportNull and reportST